src/main/java/de/tum/in/www1/artemis/web/rest/errors/ConflictException.java

```diff
@@ -0,0 +1,10 @@
+package de.tum.in.www1.artemis.web.rest.errors;
+
+import org.zalando.problem.AbstractThrowableProblem;
```

Comment:
Can we get rid of these dependencies and use a normal Java Throwable or Exception Super Class? I am not sure how stable this dependency is and whether it will be used in the future

Reply:
still valid!